### PR TITLE
Reword some stuff and add GNTP plugin

### PR DESCRIPTION
**Pull request by joshf**
Reword some notification stuff that was bugging me

Add GNTP plugin. This is needed for people with Growl from MAS. Also means users dont need growlnotify installed. This should work with windows aswell.

### CODE REVIEW

**Comment by joshf** [comment]
It should replace the Growl for user who have Growl 1.3+. Growl users with 1.2 (massively outdated) still need the growlnotify version.The new plugin _should_ work with Growl for Windows

**Comment by joshf** [comment]
I also updated the osx build stuff a bit, may aswell do it in one request

**Comment by Sbte** [comment]
Ok I'll pull/rebase this to make sure we don't have failing builds when trying to git bisect

**Comment by Sbte** [comment]
Done

File: emesene/gui/common/GNTPNotification.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with emesene; if not, write to the Free Software
+#    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+try:
+    import gntp.notifier
+except ImportError:
+    print "Sorry either Growl is not installed or GNTP Python bindings are missing..."
```
Comment (moneycat) [inline emesene/gui/common/GNTPNotification.py:22]: @joshf Maybe this message should be collected by logger. You can follow Indicator or MessagingMenu to achieve it.

Comment (moneycat) [inline emesene/gui/common/GNTPNotification.py:22]: Then it now will show in extension but I didn't install Growl or GNTP. Should it hidden when user has no backends?

**Comment by joshf** [comment]
Yeah it should as it requires growl to be installed. If it not there it wont work.

**Comment by moneycat** [comment]
Maybe you can use the import as a test in **init**.py?

**Comment by joshf** [comment]
Surely

``` python
import logging
log = logging.getLogger("gui.common.GNTPNotifications")

try:
    import gntp.notifier    
except ImportError:
    log.error("GNTP is missing...")
```

In GNTP Notifications would be okay?

File: emesene/gui/common/GNTPNotification.py (hunk 1 above)
Comment (Sbte) [inline emesene/gui/common/GNTPNotification.py:22]: @moneycat a0bcdcecab24c405c03b141e07d40cc99841ca12

It is already handled by gui.common.**init**.py

**Comment by moneycat** [comment]
Ha, Sbte done the job :P

**Comment by joshf** [comment]
It need to throw some kind of error on import otherwise it will show up in prefs and not actually do anything

**Comment by Sbte** [comment]
@joshf I already fixed this about an hour ago. No worries ;)

**Comment by moneycat** [comment]
Yes. Confirmed.

**Comment by joshf** [comment]
Yeah my bad there forgot i had gntp installed for Python 2.7 and 2.6

**Pull request closed without merging**